content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Installation (Installer Beta)"
+sidebar_current: "docs-enterprise2-private-install-installer"
+---
+
+# Private Terraform Enterprise Automated Installation (Installer Beta)
+
+The installation of Private Terraform Enterprise can be completely automated, for both online and airgapped modes.  Before invoking the installer you must:
+- prepare `/etc/replicated.conf`
+- prepare an [application settings](#application-settings) file
+- copy your license file to the instance
+- download the `.airgap` bundle to the instance (Airgapped mode only)
+
+You may also need to provide additional flags (such as the instance's public and private IP addresses) in order to not be prompted for those values when running the installer.
+
+It's expected that the user is already familiar with how to do a [manual install](./install-installer.html#installation).
+
+## Application settings
+
+This file contains the values you would normally provide in the settings screen, which may be as simple as choosing the demo installation type or as complex as specifying the Postgresql connection string and S3 bucket credentials and parameters.  You will need to reference it via the `ImportSettingsFrom` property in `/etc/replicated.conf`, which will be described below.
+
+### Format
+
+The settings file is JSON formatted.  The example below is suitable for a demo installation:
+
+```json
+{
+  "hostname": {
+    "value": "terraform.example.com"
+  },
+  "installation_type": {
+    "value": "poc"
+  },
+  "capacity_concurrency": {
+    "value": 5
+  }
+}
+```
+
+### Available settings
+
+A number of settings are available to configure and tune your installation.  They are summarized below; it is expected the user will have completed a manual installation first and already be familiar with the nature of these parameters from the settings screen.
+
+The following apply to every installation:
+
+- `hostname` — (required) this is the hostname you will use to access your installation
+- `installation_type` — (required) one of `poc` or `production`
```

Comment:
Change to 'Required' to be consistent with the Terraform provider argument format.